Automattic/wp-calypso
# composite-checkout: Add Bancontact payment method

#### Changes proposed in this Pull Request

This adds the Bancontact Stripe redirect payment method to composite checkout.

#### Screenshots

Old checkout:

![old-bancontact](https://user-images.githubusercontent.com/2036909/85619951-0e6e8c00-b631-11ea-9285-e5a5349df26b.png)

New checkout (closed):

![new-bancontact-closed](https://user-images.githubusercontent.com/2036909/85619960-13cbd680-b631-11ea-8604-30aaebe96616.png)

New checkout (open):

![new-bancontact-open](https://user-images.githubusercontent.com/2036909/85619978-1a5a4e00-b631-11ea-8a2c-401c2826c456.png)

#### Testing instructions

- Apply D45443-code to force Bancontact to be available.
- Sandbox the store.
- Change your user's currency to `EUR`.
- Add a plan to your cart and visit checkout. 
- Add `?flags=composite-checkout-force` to the query string in the URL and reload the page because currently it is disabled for international currencies.
- Verify that "Bancontact" is an option displayed in the final payment step.
- Verify that you cannot pay if the "Name" field is empty.
- Fill out the "Name" field and submit the payment.
- Verify that you are redirected to a Stripe test page.
- Click to accept the payment and verify that the payment completes successfully and that the plan is purchased.
